package.json

```diff
@@ -111,10 +111,11 @@
     "uncode-insiders": "code-insiders --uninstall-extension ms-playwright.playwright"
   },
   "devDependencies": {
+    "@babel/preset-typescript": "^7.23.2",
```

Comment:
We had to add it because of:

```
playwright-vscode on  update-babel [!] is 📦 v1.0.16 via  v18.17.1 
❯ npm run build

> playwright@1.0.16 build
> npm run esbuild -- --minify


> playwright@1.0.16 esbuild
> esbuild ./src/babelBundle.ts ./src/extension.ts ./src/oopReporter.ts ./src/debugTransform.ts --bundle --outdir=out --external:vscode --external:./babelBundle --external:./debugTransform --external:./oopReporter --format=cjs --platform=node --target=ES2019 --minify

✘ [ERROR] Could not resolve "@babel/preset-typescript/package.json"

    node_modules/@babel/core/lib/config/files/module-types.js:99:40:
      99 │             const packageJson = require("@babel/preset-typescript/package.json");
         ╵                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@babel/preset-typescript/package.json" as external to exclude it from the
  bundle, which will remove this error. You can also surround this "require" call with a try/catch
  block to handle this failure at run-time instead of bundle-time.

1 error

playwright-vscode on  update-babel [!] is 📦 v1.0.16 via  v18.17.1 
```